File: libraries/dotnet/logs/src/LogBatch.cs
// This file contains log batch logic for logs src.
namespace Otel.Logs;

internal sealed class LogBatch
{
    // Logs batch.
    public LogBatch(LogEntry firstEntry)
    {
        Logs.Add(firstEntry);
    }

    public List<LogEntry> Logs { get; } = [];
}

File: libraries/java/logs/src/main/java/otel/logs/LogBatch.java
// This file contains log batch logic for OTel logs.
package otel.logs;

import java.util.ArrayList;
import java.util.List;

final class LogBatch {
  final List<LogEntry> logs = new ArrayList<>();

  LogBatch(LogEntry firstEntry) {
    logs.add(firstEntry);
  }
}
